website/source/docs/providers/aws/r/ses_domain_identity.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "aws"
+page_title: "AWS: ses_domain_identity"
+sidebar_current: "docs-aws-resource-ses-domain-identity"
+description: |-
+  Provides an SES domain identity resource
+---
+
+# aws\_ses\_domain_identity
+
+Provides an SES domain identity resource
+
+## Example Usage
+
+```
+resource "aws_ses_domain_identity" "example" {
+  domain = "example.com"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain` - (Required) The domain name to assign to SES
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `verification_token` - A code which when added to the domain as a TXT record
+  will signal to SES that the owner of the domain has authorised SES to act on
+  their behalf. The domain identity will be in state "verification pending"
+  until this is done. Find out more about verifying domains in Amazon SES in
+  the [AWS SES docs](http://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-domains.html).
```

Comment:
Oh, I see, you have the explanation here actually, that's good 👍  but I still think we should include an example of a DNS record, like Route53 record.